components/Meeting.js

```diff
@@ -2,33 +2,50 @@ import YoutubeButton from './buttons/YoutubeButton';
 import Paragraph from './typography/Paragraph';
 import TextLink from './typography/TextLink';
 import Heading from './typography/Heading';
+import { ArrowRightIcon } from '@heroicons/react/outline';
 
 export default function Meeting({ 
     name = '', 
     purpose = '',
     host = '',
     hostProfile = '',
-    youtube = '' 
+    youtube = '', 
+    bg = ''
 }) {
 
   return (
-    <div className="rounded-md border border-gray-200 overflow-hidden bg-white p-4">
-        <Heading level="h2" typeStyle="heading-md-semibold">
-            { name }
-        </Heading>   
-        <Paragraph typeStyle="body-md" className="my-4">
-            <strong>Purpose:</strong> { purpose }
-        </Paragraph>
-        <Paragraph typeStyle="body-md" className="my-4">
-            <strong>Host:</strong> 
-            { hostProfile 
-                ? (<TextLink href={ hostProfile } target="_blank">
-                { host }
-                </TextLink>)
-                : ` ${host}.`
-            }
-        </Paragraph>
-        <YoutubeButton text="Watch recordings" href={ youtube }/>
-    </div>
+    <a href={youtube} target="_blank" rel="noreferrer">
+      <div
+        className={`meeting-card overflow-hidden p-4 bg-${bg} w-full lg:w-[300px] h-[300px] cursor-pointer hover:bg-dark hover:text-white flex flex-col justify-between`}
+      >
+        <div>
+          <h3 className="text-xl">{name}</h3>
+          <div>
+            <Paragraph typeStyle="body-sm" className="my-4" textColor="white">
+              {purpose}
+            </Paragraph>
+          </div>
+        </div>
+        <div className="flex items-center justify-between">
+          <Paragraph typeStyle="body-md" className="my-4">
+            <strong>Host:</strong>
+            {hostProfile ? (
+              <TextLink
+                href={hostProfile}
+                target="_blank"
+                className="hover:text-primary-500"
+              >
+                {host}
+              </TextLink>
+            ) : (
+              ` ${host}.`
+            )}
+          </Paragraph>
+          <div>
+            <ArrowRightIcon className="w-[20px] ml-3 mt-1 text-slate-400" />
```

Comment:
```suggestion
            <ArrowRightIcon className="w-[20px] ml-3 text-slate-400" />
```

I know removing the margin-top will not completely make it center but still, it will align with the text beside it.